qutip/visualization.py

```diff
@@ -170,6 +171,199 @@ def _cb_labels(left_dims):
     ]
 
 
+# Adopted from the SciPy Cookbook.
+def new_hinton(rho, xlabels=None, ylabels=None, label_top=True, color_style="scaled",
+           figure = None, figure_options = {}, axes = None, axes_options = {}):
```

Comment:
Hi. I changed arguments. 
Figure_options gives figsize, dpi, etc... to a Figure object and axes_options works almost the same as figure_options do, but beginners may not know which contains the title argument. 
Hence, I deleted them because you can preset these options to Figure and Axes objects. Instead, I added x and y labels, title and cmap (also figsize?). Most people are satisfied with that, I think.